framework/utils/InputData.py

```diff
@@ -495,10 +496,9 @@ def addSub(cls, sub, quantity=Quantity.zero_to_infinity):
     cls.subs.add(sub)
     if cls.subOrder is not None:
       cls.subOrder.append((sub,quantity))
-    # I did not see the value of the following check --- wangc
-    #elif quantity != Quantity.zero_to_infinity:
-    #  print("ERROR only zero to infinity is supported if Order==False ",
-    #        sub.getName()," in ",cls.getName())
+    elif quantity != Quantity.zero_to_infinity:
+     print("ERROR only zero to infinity is supported if Order==False ",
```

Comment:
Why do we have an error print instead of raising an error?

Reply:
I think the error print is only for XSD, which means if we have this input, when we start to generate the XSD using InputData, we will have the violation. I think the print is for this purpose. Josh please correct me if I'm wrong.

Reply:
@joshua-cogliati-inl

Reply:
This only becomes a problem if we try and generate an XSD, so at this point, an error would probably mostly be an annoyance.  @wangcj05  Did you add an explicit check for quantity in the InputData (Note that adding that is not required for this merge, but if you didn't an issue probably would be useful.)?

Reply:
I haven't add the check, and I will open an issue for that

Reply:
See #967

Reply:
Thank you, that's good for me. This might be better framed as a `warning` if it isn't a proper error, if you happen to be making other changes and want to change this as well.